src/commands/CommandMusicPlay.java

```diff
@@ -138,7 +166,8 @@ public String getCommandDescription(){
 	public ParametersHelp[] getParametersDescriptions(){
 		return new ParametersHelp[]
 		{
-			new ParametersHelp(lang("ReplayDescription"), "l", "latest")
+			new ParametersHelp(lang("ReplayDescription"), "l", "latest"),
+			new ParametersHelp(lang("AnythingDescription"), "a", "anything")
```

Comment:
Well, issue #44 says the use of `--anything` but it feels wrong to use the parameter `a`, it makes it seems like you were playing `all` the music.

I'd suggest using instead `-r` and `--random`, what do you think?